tests/test_cli.py

```diff
@@ -106,6 +106,6 @@ def test_cli_with_lang_to(self):
         output = self.cli.run_cmd(150.55, '--lang', 'es', '--to', 'currency')
         self.assertEqual(output.return_code, 0)
         self.assertEqual(
-            output.out.strip(),
-            "ciento cincuenta euros con cincuenta y cinco centimos"
+            output.out.strip().decode('utf-8'),
```

Comment:
Would it make sense to add `locale.getpreferredencoding()` here instead of `utf-8`?